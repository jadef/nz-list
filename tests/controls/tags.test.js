import React from 'react';
import renderer from 'react-test-renderer';

// Link to component(s)
import ControlsTags from '../../app/js/components/controls/Tags';
import Tag from '../../app/js/components/controls/Tags';

// Describe
test('ControlsTags Renders', () => {
  const component = renderer.create(
    <ControlsTags />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


test('Tag Renders', () => {
  const component = renderer.create(
    <Tag />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
