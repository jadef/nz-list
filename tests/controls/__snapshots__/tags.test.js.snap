// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ControlsTags Renders 1`] = `
<section
  className="tags controls null"
>
  <ul
    className="list"
  >
    <li
      className={null}
      onClick={[Function]}
    >
      Animation
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Abstraction
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Automation
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      CMS
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      CSS
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Framework
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      JavaScript
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Library
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Manager
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Mobile
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Node
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      PHP
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Preprocessor
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Ruby
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Single Page
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Templating
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Testing
    </li>
  </ul>
  <div
    className="trigger"
  >
    <a
      className={null}
      href="#"
      onClick={[Function]}
    >
      Tags
    </a>
  </div>
</section>
`;

exports[`Tag Renders 1`] = `
<section
  className="tags controls null"
>
  <ul
    className="list"
  >
    <li
      className={null}
      onClick={[Function]}
    >
      Animation
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Abstraction
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Automation
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      CMS
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      CSS
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Framework
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      JavaScript
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Library
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Manager
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Mobile
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Node
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      PHP
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Preprocessor
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Ruby
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Single Page
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Templating
    </li>
    <li
      className={null}
      onClick={[Function]}
    >
      Testing
    </li>
  </ul>
  <div
    className="trigger"
  >
    <a
      className={null}
      href="#"
      onClick={[Function]}
    >
      Tags
    </a>
  </div>
</section>
`;
